pkg/virt-controller/watch/vmi/vmi.go

```diff
@@ -661,6 +662,9 @@ func (c *Controller) updateStatus(vmi *virtv1.VirtualMachineInstance, pod *k8sv1
 			c.syncVolumesUpdate(vmiCopy)
 		}
 
+		// Requires some action in the VM to trigger this check, is this ok?
```

Comment:
What do you mean by that?

Reply:
In the scenario in which the VM is running, and the user disables the feature by editing the KubeVirt CR, the VMI object is not instantly reconciliated, it takes some time. I'm not sure if that's ok or should we trigger a VMI update somehow.